File: nodejs/src/simple-interval.js
const ticktok = require('ticktok');

const domain = process.env.TICKTOK_DOMAIN || 'http://localhost:9643';
const token = process.env.TICKTOK_TOKEN || 'ticktok-zY3wpR';

ticktok(domain, token)
  .schedule(
    { name: 'simple-interval', schedule: 'every.10.seconds' },
    (err, tick) => { console.log('tick') }
  );

File: java/src/main/java/io/ticktok/example/SimpleInterval.java
package io.ticktok.example;

import io.ticktok.client.Ticktok;

import static io.ticktok.client.Ticktok.options;

public class SimpleInterval {

    public static void main(String[] args) {
        new Ticktok(options().domain(args[0]).token(args[1]))
                .schedule("simple-interval-java", "every.30.seconds", () -> System.out.println("tick"));
    }
}
